esmvalcore/cmor/check.py

```diff
@@ -426,6 +426,16 @@ def _check_dim_names(self):
                                 coordinate.out_name,
                             )
                             coord.var_name = coordinate.out_name
+                        elif coord.standard_name in ['region', 'area_type']:
+                            self.report_debug_message(
+                                'Coordinate {0} has var name {1} '
+                                'instead of {2}. '
+                                "But that's considered OK and ignored. "
+                                'See also https://github.com/ESMValGroup/ESMValCore/discussions/1587',  # noqa: line-too-long
```

Comment:
```suggestion
                                'But that's considered OK and ignored. See also '
                                'https://github.com/ESMValGroup/ESMValCore/discussions/1587',
```
maybe you could try if this works? Usually if you put the URL on it's own line the linter does not complain

Reply:
I tried. Still too long and flake doesn't like it either.

Reply:
I added the link as a separate variable at the beginning of the function. Maybe it's not the cleanest thing to do but the code in the checker is quite idented. I will merge this if all tests pass.